src/common/types/tasks.js

```diff
@@ -8,6 +8,7 @@ import type {
     MarkerPaginatedCollection,
     SelectorItems,
 } from './core';
+import { FEED_ITEM_TYPE_TASK } from '../../constants';
```

Comment:
Can we revert? Common types should be the source of truth, rather than using Feed in them.